### PR TITLE
Implemented Disable Action Prop for SBC-Header

### PR DESCRIPTION
**Pull request by cameron-freshworks**
* Implemented disable prop for SBC Header Actions

*Notes:*

* In Name Request, we wanted to hide the actions as a whole until the functionality is ready on the Namex Side.

### CODE REVIEW

File: vue/sbc-common-components/src/components/SbcHeader.vue [hunk 1]
```diff
@@ -14,7 +14,7 @@
         </picture>
         <span class="brand__title">BC Registries <span class="brand__title--wrap">and Online Services</span></span>
       </a>
-      <div class="app-header__actions">
+      <div v-if="!disableActions" class="app-header__actions">
```
Comment (severinbeauvais) [inline vue/sbc-common-components/src/components/SbcHeader.vue:17]: Double negative is confusing. Easy fix?

Comment (severinbeauvais) [reply vue/sbc-common-components/src/components/SbcHeader.vue:17]: With a default of true?

**Review by severinbeauvais — APPROVED** [review]

File: vue/sbc-common-components/src/components/SbcHeader.vue (hunk 1 above)
Comment (cameron-freshworks) [reply vue/sbc-common-components/src/components/SbcHeader.vue:17]: Hmm, yea lol. 
I'll change the prop title to `showActions` defaulted to `true`

**Review by severinbeauvais — APPROVED** [review]
Nice!

**Pull request merged**